blog/2017-02-03-using-native-driver-for-animated.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Using Native Driver for Animated
+author: Janic Duplessis
+authorTitle: Software Engineer at App & Flow
+authorURL: https://twitter.com/janicduplessis
+authorImage: https://secure.gravatar.com/avatar/8d6b6c0f5b228b0a8566a69de448b9dd?s=128
+authorTwitter: janicduplessis
+category: engineering
+---
+
+For the past year, we've been working on improving performance of animations that use the Animated library. Animations are very important to create a beautiful user experience but can also be hard to do right. We want to make it easy for developers to create performant animations without having to worry about some of their code causing it to lag.
+
+## What is this?
+
+The Animated API was designed with a very important constraint in mind, it is serializable. This means we can send everything about the animation to native before it has even started and allows native code to perform the animation on the UI thread without having to go through the bridge on every frame. It is very useful because once the animation has started, the JS thread can be blocked and the animation will still run smoothly. In practice this can happen a lot because user code runs on the JS thread and react renders can also lock JS for a long time.
+
+## A bit of history...
+
+This project started about a year ago, when Exponent built the li.st app on Android. [Krzysztof Magiera](https://twitter.com/kzzzf) was contracted to build the initial implementation on Android. It ended up working well and li.st was the first app to ship with native driven animations using Animated. A few months later [Brandon Withrow](https://github.com/buba447) built the initial implementation on iOS and after that [Ryan Gomba](https://twitter.com/ryangomba) and myself worked on adding missing features like support for `Animated.event` as well as squash bugs we found when using it in production apps. This was truly a community effort and I would like to thanks everyone that was involved as well as Exponent for sponsoring a large part of the development. As of today it is now used by `Touchable` components in React Native as well as for navigation animations in the newly released [React Navigation](https://github.com/react-community/react-navigation) library.
```

Comment:
You can add comma after "As of today" or remove that phrase. The following works just as well:

> It is now used by `Touchable` components in React Native as well as for navigation animations in the newly released [React Navigation](https://github.com/react-community/react-navigation) library.